common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -14,10 +14,9 @@
 
 define(
 'video/01_initialize.js',
-['video/03_video_player.js', 'video/00_i18n.js'],
-function (VideoPlayer, i18n) {
-    var moment = window.moment;
-
+['video/03_video_player.js', 'video/00_i18n.js', 'moment'],
+function (VideoPlayer, i18n, momentjs) {
+    var moment = momentjs || window.moment;
```

Comment:
@waheedahmed I'm still not sure about this as it was giving error randomly on your devstack. Also i could't understand why the `window.moment`  gets undefined somewhere.  Did you know why the `window.moment ` gets undefined in existing code.